test/y2storage/proposal/autoinst_devices_creator_test.rb

```diff
@@ -417,6 +432,8 @@
           expect(result.shrinked_partitions.map(&:planned)).to eq([root])
         end
       end
+
+      context "when "
```

Comment:
when?

Reply:
Eeerrr never. I was about to add the test there, but I found out that we already were testing that scenarion. Fixed.